test/test_core.py

```diff
@@ -5781,6 +5781,7 @@ def test_fs_write(self):
   def test_fs_emptyPath(self):
     self.do_run_in_out_file_test('fs/test_emptyPath.c')
 
+  @no_windows('https://github.com/emscripten-core/emscripten/issues/8882')
```

Comment:
Can you add `@crossplatform` here too?  That way when we fix this all we do is remove the `@no_windows`